examples/scripts/my_test.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python
+
+# Copyright: (c) 2018, Terry Jones <terry.jones@example.org>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: my_test
+
+short_description: This is my test module
+
+# If this is part of a collection, you need to use semantic versioning,
+# i.e. the version is of the form "2.5.0" and not "2.4".
+version_added: "2.4"
```

Comment:
```suggestion
version_added: "1.0.0"
```

Reply:
since the majority of new modules will be in collections, let's have the examples make it obvious that we don't use Ansible versions anymore.